src/docs/product/data-management-settings/scrubbing/server-side-scrubbing.mdx

```diff
@@ -43,7 +43,7 @@ credentials = {
 }
 ```
 
-Using the default options would not cause the _object_ `credentials` to be redacted in its entirety; rather, all of its entries would be subject to scrubbing. So `password` would be redacted by default, and adding `cats`, `username`, and/or `lastLogin` to the list of additional fields would cause those values to be redacted as well.
+Using the default options would cause the _object_ `credentials` to be set to `null`, and the object itself will not be entered. To prevent this from happening the keyname can be added to Safe Fields, which will leave the `credentials` object intact with all its contents.
```

Comment:
Did you verify this? I am not 100% sure if putting `credentials` in `Safe Fields` will leave the entire object untouched, or if it will descend and scrub the `password` field anyway.

Reply:
Added test in https://github.com/getsentry/relay/pull/1789/commits/05caa3442f50a2fe3b3b65a21d4a752504905a84 to verify. Also updated the doc accordingly.

Reply:
Thanks for double checking!